gasmodel/Pact/Core/GasModel/InterpreterGas.hs

```diff
@@ -1300,7 +1300,7 @@ gasContType pdb = \case
     gasLetC pdb
   CTSeqC ->
     gasSeqC pdb
-  CTListC -> undefined
+  CTListC -> C.bgroup "list" []
```

Comment:
We'll add more benchmarks later, but for now `undefined` just breaks running all the benchmarks or matching on a pattern.